front/src/TurtleEnableBar.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import TurtleEnableButton from "./TurtleEnableButton";
+import styled from "styled-components";
+
+/**
+ * Simple bar with all the turtles
+ * Author: S.A. Tanja
+ * Author: H.E. van der Laan
+ */
+
+const TurtleEnableBar = props => {
+  return (
+    <div className={props.className}>
```

Comment:
Why do you need this `div`?

Reply:
You need a div if you return multiple elements